INSTALL.md

```diff
@@ -1,11 +1,14 @@
 Installing From Sources
 =======================
 
-This document presents instructions to install this branch of Coq.
-For more general installation instructions and information about known
-build system issues, please consult the wiki page: 
+The recommended way to install and use Coq is to install the Coq
+platform or to rely on a package manager (e.g. opam or Nix).
```

Comment:
```suggestion
We recommend using a package manager such as opam or Nix to install the Coq platform.
```

Reply:
This changes the meaning completely, so nack.

Reply:
Yeah, I missed a bit of the original meaning.  My suggestion was mainly to begin with "We recommend ..."